go.mod

```diff
@@ -3,58 +3,63 @@ module github.com/aws/amazon-vpc-cni-k8s
 go 1.12
 
 require (
```

Comment:
What is the actual dependency change? How easy is this PR to cherry pick over to the `release-1.4` branch?